patches/pe-477-scheduler-content-moderation-integration.patch

```diff
@@ -0,0 +1,247 @@
+From 8a7a14603f6091ae5b46e821d2aa4a72078db780 Mon Sep 17 00:00:00 2001
```

Comment:
Is this file used anywhere? It looks like composer now loads the patch remotely instead of from a local file.

Reply:
Good catch, nope not needed. I copied in the new patch to test but forgot to delete the local file after I switched over to the drupal.org one.